frontend/src/metabase/visualizations/visualizations/SmartScalar/SmartScalar.jsx

```diff
@@ -248,36 +270,60 @@ function PreviousValueComparison({
       }) <= availableComparisonWidth,
   );
 
-  const VariationPercent = ({ iconSize, children }) => (
-    <Variation color={changeColor}>
-      {changeArrowIconName && (
-        <VariationIcon name={changeArrowIconName} size={iconSize} />
-      )}
-      <VariationValue showTooltip={false}>{children}</VariationValue>
-    </Variation>
-  );
-  const VariationDetails = ({ children }) =>
-    children ? (
-      <PreviousValueDetails>
+  const VariationPercent = ({ inTooltip, iconSize, children }) => {
+    const noChangeColor =
+      inTooltip || isNightMode
+        ? lighten(theme.fn.themeColor("text-medium"), 0.3)
+        : "text-light";
+
+    return (
+      <Flex align="center" maw="100%" c={changeColor ?? noChangeColor}>
+        {changeArrowIconName && (
+          <VariationIcon name={changeArrowIconName} size={iconSize} />
+        )}
+        <VariationValue showTooltip={false}>{children}</VariationValue>
+      </Flex>
+    );
+  };
+
+  const VariationDetails = ({ inTooltip, children }) => {
+    if (!children) {
+      return null;
+    }
+
+    const detailColor =
+      isNightMode || inTooltip
+        ? lighten(theme.fn.themeColor("text-light"), 0.25)
+        : "text-medium";
+
+    return (
+      <Title order={4} c={detailColor} style={{ whiteSpace: "pre" }}>
         {separator}
         {children}
-      </PreviousValueDetails>
-    ) : null;
+      </Title>
+    );
+  };
 
   return (
     <Tooltip
       isEnabled={fullDetailDisplay !== fittedDetailDisplay}
       placement="bottom"
       tooltip={
-        <VariationContainerTooltip className="variation-container-tooltip">
```

Comment:
we don't need this class name anymore as we apply it directly using `inTooltip` below